siac/rentercmd.go

```diff
@@ -353,9 +398,15 @@ func renterfilesdownloadcmd(path, destination string) {
 // SiaPath field.
 type bySiaPath []modules.FileInfo
 
-func (s bySiaPath) Len() int           { return len(s) }
-func (s bySiaPath) Swap(i, j int)      { s[i], s[j] = s[j], s[i] }
-func (s bySiaPath) Less(i, j int) bool { return s[i].SiaPath < s[j].SiaPath }
+func (s bySiaPath) Len() int {
+	return len(s)
+}
+func (s bySiaPath) Swap(i, j int) {
+	s[i], s[j] = s[j], s[i]
+}
+func (s bySiaPath) Less(i, j int) bool {
+	return s[i].SiaPath < s[j].SiaPath
+}
```

Comment:
Oh, this must have been `gofmt`. Is there a way to run `gofmt` only on the changes made?